src/Compilers/Core/Portable/Operations/CommonConversion.cs

```diff
@@ -50,6 +52,10 @@ internal CommonConversion(bool exists, bool isIdentity, bool isNumeric, bool isR
         /// </summary>
         public bool IsIdentity => (_conversionKind & ConversionKind.IsIdentity) == ConversionKind.IsIdentity;
         /// <summary>
+        /// Returns true if the conversion is an nullable conversion.
+        /// </summary>
+        public bool IsNullable => (_conversionKind & ConversionKind.IsNullable) == ConversionKind.IsNullable;
```

Comment:
this seems safe to add to the common layer.  Both VB and C# have htis concept and it seems identical from my reading.  Namely:

c#:

> Implicit nullable conversions
>
> Predefined implicit conversions that operate on non-nullable value types can also be used with nullable forms of those types. For each of the predefined implicit identity and numeric conversions that convert from a non-nullable value type S to a non-nullable value type T, the following implicit nullable conversions exist:
>
> * An implicit conversion from S? to T?.
> * An implicit conversion from S to T?.
>
> Evaluation of an implicit nullable conversion based on an underlying conversion from S to T proceeds as follows:
>
> * If the nullable conversion is from S? to T?:
>   * If the source value is null (HasValue property is false), the result is the null value of type T?.
>   * Otherwise, the conversion is evaluated as an unwrapping from S? to S, followed by the underlying conversion from S to T, followed by a wrapping (Nullable types) from T to T?.
>
> * If the nullable conversion is from S to T?, the conversion is evaluated as the underlying conversion from S to T followed by a wrapping from T to T?.

(and the corresponding explicit version of this)

VB:

> A value type T can convert to and from the nullable version of the type, T?. The conversion from T? to T throws a System.InvalidOperationException exception if the value being converted is Nothing. Also, T? has a conversion to a type S if T has an intrinsic conversion to S. And if S is a value type, then the following intrinsic conversions exist between T? and S?:
>
> * A conversion of the same classification (narrowing or widening) from T? to S?.
> * A conversion of the same classification (narrowing or widening) from T to S?.
> * A narrowing conversion from S? to T.